src/ui/page-issue-link.tsx

```diff
@@ -8,7 +8,14 @@ export const PageIssueLink: Component = () => {
 	const i18n = useI18n();
 
 	const srcPath = createMemo(() => {
-		return `https://github.com/solidjs/solid-docs-next/issues/new?assignees=ladybluenotes&labels=improve+documentation%2Cpending+review&projects=&template=CONTENT.yml&title=[Content]:&subject=${getEntryFileName()}`;
+		return `https://github.com/solidjs/solid-docs-next/issues/new` +
+			`?assignees=ladybluenotes` +
+			`&labels=improve+documentation%2Cpending+review` +
+			`&projects=` +
+			`&template=CONTENT.yml` +
+			`&title=[Content]:` +
+			`&subject=${getEntryFileName()}` +
+			`&page=${window.location.href}`
 	});
```

Comment:
@brenelz A formatting issue is blocking this PR. Could you please format the file or use the following code:

```tsx
	const srcPath = createMemo(() => {
		return (
			"https://github.com/solidjs/solid-docs-next/issues/new" +
			"?assignees=ladybluenotes" +
			"&labels=improve+documentation%2Cpending+review" +
			"&projects=" +
			"&template=CONTENT.yml" +
			"&title=[Content]:" +
			"&subject=${getEntryFileName()}" +
			"&page=${window.location.href}"
		);
	});
```

Reply:
Hi @amirhhashemi , this should be resolved now!

Reply:
@bertday My bad, I should've used \` instead of \" for strings. I didn't notice template strings are required.

Reply:
Hi @amirhhashemi , just pushed a change to bring this backticks back. Really hoping this does this trick.